MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/LogoutController.java

```diff
@@ -0,0 +1,41 @@
+package com.softserve.if072.mvcapp.controller;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+
+@Controller
+@RequestMapping("/logout")
+@PropertySource(value = {"classpath:application.properties"})
+public class LogoutController {
+
+    private static final Logger LOGGER = LogManager.getLogger(LogoutController.class);
+    private final String AUTHENTICATION_COOKIE_NAME;
+
+    private Environment environment;
```

Comment:
Do you need this?
Please chec if you can use @Value annotation on the class field. Foe xample:
`@Value("application.authenticationCookieName")`
`private String cookieName;`